samples/cpp/chat_sample/chat_sample.cpp

```diff
@@ -10,7 +10,7 @@ int main(int argc, char* argv[]) try {
     std::string prompt;
     std::string models_path = argv[1];
 
-    std::string device = "CPU";  // GPU, NPU can be used as well
+    std::string device = "NPU";  // GPU, NPU can be used as well
```

Comment:
I believe the default device should remain `CPU`

Reply:
Fixed